README.md

```diff
@@ -1,11 +1,28 @@
 # moloch-monorepo
 
 **install ganache-cli**
-1. npm install -g ganache-cli
-2. ganache-cli -m \"fetch local valve black attend double eye excite planet primary install allow\" -a 100
+  
+  Ganache can create a personal blockchain for us that we can use it create accounts and do tests easier.+
+  To install it we use npm like that:
+  
+1. `npm install -g ganache-cli`
+2. `ganache-cli -h 0.0.0.0 -m \"fetch local valve black attend double eye excite planet primary install allow\" -a 100`
```

Comment:
I've never had to do this `-h` command, is there a reason you had to do it?

Reply:
I couldn't connect the graph node with the ganache.